department-of-veterans-affairs/va.gov-cms
# VACMS-2285 facility services

## Description
See #2285 

## Testing done
Visual, behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/88349219-77731d80-cd1d-11ea-9c5b-c25adc0163ba.png)
![image](https://user-images.githubusercontent.com/2404547/88349227-7cd06800-cd1d-11ea-8a89-6f87cb8d7c24.png)

## QA steps
- As an admin, go to `/node/350/edit` and visually verify that you see an `Add service location` button
- Click on it, and confirm it brings up three fieldsets: Address, phone, hours
- Enter data into each fieldset, create a second instance of service location, and enter data in it.
- Save page, and visually verify that all data is available.
